src/builder/pod/mod.rs

```diff
@@ -446,7 +504,34 @@ mod tests {
                 topology_key: "topology.kubernetes.io/zone".to_string(),
                 ..Default::default()
             }]),
-        };
+        }
+    }
+
+    #[fixture]
+    fn pod_anti_affinity(pod_affinity: PodAffinity) -> PodAntiAffinity {
+        PodAntiAffinity {
+            preferred_during_scheduling_ignored_during_execution: None,
+            required_during_scheduling_ignored_during_execution: pod_affinity
+                .required_during_scheduling_ignored_during_execution,
+        }
+    }
+
+    #[rstest]
+    fn test_pod_builder(pod_affinity: PodAffinity) {
+        let container = ContainerBuilder::new("containername")
+            .expect("ContainerBuilder not created")
+            .image("stackable/zookeeper:2.4.14")
+            .command(vec!["zk-server-start.sh".to_string()])
+            .args(vec!["stackable/conf/zk.properties".to_string()])
+            .add_volume_mount("zk-worker-1", "conf/")
+            .build();
+
+        let init_container = ContainerBuilder::new("init-containername")
+            .expect("ContainerBuilder not created")
+            .image("stackable/zookeeper:2.4.14")
+            .command(vec!["wrapper.sh".to_string()])
+            .args(vec!["12345".to_string()])
+            .build();
```

Comment:
I think only container name and image are required for a dummy test.

Reply:
that code was already there, I didn't add it (The diff is wrong). But I can still make those changes if you want to

Reply:
I cleaned it up a bit: 9af360a